### PR TITLE
Replace Gitter with Slack

### PR DESCRIPTION
**Pull request by jpkrohling**
Signed-off-by: Juraci Paixão Kröhling <juraci@kroehling.de>

### CODE REVIEW

File: content/docs/1.22/_index.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ children:
 
 Welcome to Jaeger's documentation portal! Below, you'll find information for beginners and experienced Jaeger users.
 
-If you can't find what you are looking for, or have an issue not covered here, we'd love to hear from you either on [Gitter chat](https://gitter.im/jaegertracing/Lobby), our [mailing list](https://groups.google.com/forum/#!forum/jaeger-tracing) or [Github](https://github.com/jaegertracing/jaeger/issues).
+If you can't find what you are looking for, or have an issue not covered here, we'd love to hear from you either on [CNCF Slack chat](https://cloud-native.slack.com/archives/CGG7NFUJ3), our [mailing list](https://groups.google.com/forum/#!forum/jaeger-tracing) or [Github](https://github.com/jaegertracing/jaeger/issues).
```
Comment (yurishkuro) [inline content/docs/1.22/_index.md:11]: How about just pointing to get-in-touch page?

File: content/get-in-touch.md [hunk 2]
```diff
@@ -4,7 +4,7 @@ title: Getting in touch
 
 ## Via chat or email
 
-Join our open online chat room [gitter.im/jaegertracing](https://gitter.im/jaegertracing/Lobby) or Google mail group [jaeger-tracing@googlegroups.com](https://groups.google.com/forum/#!forum/jaeger-tracing).
+Join our open online chat room [#jaeger at the CNCF Slack](https://cloud-native.slack.com/archives/CGG7NFUJ3Lobby) or Google mail group [jaeger-tracing@googlegroups.com](https://groups.google.com/forum/#!forum/jaeger-tracing).
```
Comment (yurishkuro) [inline content/get-in-touch.md:7]: The direct link is useful, but we also need to give the sign-up link.

File: themes/jaeger-docs/layouts/partials/footer.html [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {{ $githubRepo    := .Site.Params.githubrepo }}
 {{ $twitterHandle := .Site.Params.twitterhandle }}
-{{ $gitterHandle  := .Site.Params.gitterhandle }}
+{{ $chatHandle  := .Site.Params.chathandle }}
```
Comment (yurishkuro) [inline themes/jaeger-docs/layouts/partials/footer.html:3]: Var not defined in config and not used here

File: content/get-in-touch.md (hunk 2 above)
Comment (jpkrohling) [reply content/get-in-touch.md]: Done

File: themes/jaeger-docs/layouts/partials/footer.html (hunk 3 above)
Comment (jpkrohling) [reply themes/jaeger-docs/layouts/partials/footer.html]: Oops

File: content/docs/1.22/_index.md (hunk 1 above)
Comment (jpkrohling) [reply content/docs/1.22/_index.md]: Good idea

**Review by yurishkuro — APPROVED** [review]

**Pull request merged**